utilities/blob_db/blob_compaction_filter.cc

```diff
@@ -295,16 +398,24 @@ BlobIndexCompactionFilterFactory::CreateCompactionFilter(
 
   assert(blob_db_impl());
 
-  BlobCompactionContext context;
-  blob_db_impl()->GetCompactionContext(&context);
+  BlobCompactionContext bcontext;
+  blob_db_impl()->GetCompactionContext(&bcontext);
```

Comment:
Nit: please either leave the name as-is, or rename to `blob_compaction_context`. But personally I prefer the shorter `context`. Also affects the other class below.

Reply:
Same here, use the name context would have build warning, use _context instead.